### DISCUSSION TITLE
Issue with PHP Image: Missing ICU Library (libicuio.so.70)

### DISCUSSION THREAD

**Discussion by Naveen-is-here** [wolfi-dev/os#30036] [Q&A]
Hi Wolfi Team,

I’m running into an issue when building a PHP image through wolfi. I’m getting the following error:

**php: error while loading shared libraries: libicuio.so.70: cannot open shared object file: No such file or directory**

It seems related to the ICU libraries in the image. I’m still not sure how to tackle this. Even creating a symlink didn’t resolve the issue.

Could you provide some guidance or help on how to fix this?

Thanks!

**Comment by xnox** [comment]
> php: error while loading shared libraries: libicuio.so.70: cannot open shared object file: No such file or directory

In wolfi, icu is already at version 75.1. When installing `php-8.3-intl` it is linked against `/usr/lib/libicuio.so.75`.

```
# ldd /usr/lib/php/modules/intl.so | grep icuio
	libicuio.so.75 => /usr/lib/libicuio.so.75 (0x000077874632c000)
```
